src/dc_securejoin.rs

```diff
@@ -262,7 +268,13 @@ pub unsafe fn dc_join_securejoin(
                     );
                 }
                 // Bob -> Alice
-                while !(0 != *context.shall_stop_ongoing.clone().read().unwrap()) {
+                while !(context
+                    .running_state
+                    .clone()
+                    .read()
+                    .unwrap()
+                    .shall_stop_ongoing)
```

Comment:
Maybe refactor these (don't have to do it now) into a method? E.g. `shall_stop_ongoing(context)`

Reply:
yeah :D these are not nice to read